.github/workflows/i18n-crowdin-download.yml

```diff
@@ -0,0 +1,53 @@
+name: Crowdin Download Action
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 * * * *" # TODO: Do we want to run this every hour?
+
+jobs:
+  download-sources-from-crowdin:
+    runs-on: ubuntu-latest
+
+# TODO: For the test repo this is not required. Maybe for grafana/grafana?
+#    permissions:
+#      contents: write # needed to commit changes back into the PR
+#      pull-requests: write # needed to update PR description
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Download sources
+        uses: crowdin/github-action@v1
+        with:
+          upload_sources: false
+          upload_translations: false
+          download_sources: false
+          download_translations: true
+          localization_branch_name: l10n_crowdin_translations
+          create_pull_request: true
+          pull_request_title: 'I18n: Download translations from Crowdin'
+          pull_request_body:  |
+            :robot: Automatic download of translations from Crowdin.
+
+            Steps for merging:
+              1. Wait for the **New Crowdin translations by GitHub Action** commit that may be required for CI to pass.
+              2. A quick sanity check of the changes and approve. Things to look out for:
+                - No changes in the English file. The source of truth is in the main branch, NOT in Crowdin.
+                - Translations maybe be removed if the English phrase was removed, but there should not be many of these
+                - Anything else that looks 'funky'. Ask if you're not sure.
+              3. Approve & (Auto-)merge. :tada:
+
+            If there's a conflict, close the pull request and **delete the branch**. A GH action will recreate the pull request.
+            Remember, the longer this pull request is open, the more likely it is that it'll get conflicts.
+          pull_request_labels: 'area/frontend, area/internationalization, no-changelog, no-backport'
+          pull_request_reviewers: 'grafana-frontend-platform'
+          pull_request_base_branch_name: 'main'
+          base_url: 'https://grafana.api.crowdin.com'
+          config: 'crowdin.yml'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
```

Comment:
We'll want to run our formatting/processing on the files that crowdin downloads:

 - Change the crowdin/github-action action to not commit or create the pull request
 - Add a step to run `yarn i18n:extract`
 - Use something like https://github.com/marketplace/actions/create-pull-request to create the pull request instead

See this thread https://github.com/crowdin/github-action/issues/178#issuecomment-1602171816

Reply:
Oh, hmm, perhaps this isn't required due you to removing other languages from the i18next-parser file